### ISSUE TITLE
Problem importing scvi using reticulate

### ISSUE THREAD

**Issue by stmartineau** [scverse/scvi-tools#1528]
I have set up a conda environment for scvi-tools and I'm trying to import packages into R using reticulate. Scanpy imports fine, but scvi-tools doesn't. I've tried creating the environment from within R and in the terminal, but neither works.

This is in Windows. First I tried it on my Mac, but I had an issue where the R session kept aborting when I loaded the scvi environment. After searching Stack Exchange for solutions, I gave up on solving that issue so I tried it on the Windows computer but now I'm having a different problem.

```R
library(reticulate) 

# create the env from within R
conda_create(envname = "scvi_env", packages = c("r-reticulate", "scanpy", "scvi-tools"))
# in the terminal, conda activate scvi-env

# use the correct python
use_python("C:/Users/sm607/Anaconda3/envs/scvi-env/python.exe")

# load the env
use_condaenv("scvi_env", required = T)

# scanpy imports, but not scvi
sc <- import("scanpy", convert = FALSE)
scvi <- import("scvi", convert = FALSE)
```

```pytb
Show in New Window
 Error in py_module_import(module, convert = convert) : 
ImportError: cannot import name 'get_num_classes' from 'torchmetrics.utilities.data' (C:\Users\sm607\ANACON~1\envs\scvi_env\lib\site-packages\torchmetrics\utilities\data.py)
```

#### Versions:

> VERSION
scvi-tools-0.14.6

**Comment by adamgayoso** [comment]
Unfortunately, I'm not sure scvi-tools is supported on windows easily due to a new jax dependency introduced in 0.15.0. You would have to build jax from source I believe for windows.

Mac would be your best bet, but if you're using a new M1 mac, you have to use a native version of miniconda (designed for arm64) and not via rosetta.

**Comment by stmartineau** [comment]
> 

That's good to know, thanks. I have an Intel-based Mac. I created a conda environment using this command:

conda create -n scvi-env -c conda-forge scanpy scvi-tools numpy=1.21.4 numba=0.53 python=3.9

This originally worked for me. But a week later it suddenly stopped working because whenever I tried to run `use_condaenv("scvi_env", required = T)` in R, the R session aborted. I tried to make sure it was using the right python with `py_config` and `use_python` but nothing worked for me.
